ccsm/baseline-ccsm-v1.0-psd01.md

```diff
@@ -216,25 +216,37 @@ A CCSM utilized in a BPI MUST support cryptographic algorithms based on commonly
 
 The usage of cryptographic libraries that successfully passed the National Institute of Standards and Technology (NIST) Cryptographic Module Verification Program (CMVP) is recommended.
 
+[[R1]](#r1) testability: Any security-audited cryptographic library will have test-vectors, and is, therefore, testable.
+
 #### **[R2]**	
 If the CCSM utilized in a BPI utilizes a Peer-to-Peer (P2P) message protocol, the protocol MUST support end-to-end encryption.
 
+[[R2]](#r2) testability: libp2p as a very commonly used P2P message protocol supports end-to-end encryption, see [here](https://github.com/libp2p/specs/blob/master/noise/README.md) for the NOISE specification and relevant tests [here](https://github.com/libp2p/go-libp2p/blob/master/test-plans/cmd/ping/main.go). This shows that the requirement is testable. 
```

Comment:
This show of testability seems to hinge upon libp2p being used. Does this not create further requirement for implementers to use libp2p? Why can we not just hint that outside protocols implementing test suites constitutes testability?

Reply:
an example with a test suite that meets and tests the requirement is sufficient. there does not have to be a formal standard defined with a test suite for reference implementations. One could argue abstractly but concretely is better IMHO. @Kasshern

Reply:
In that case, for me as well, LGTM